src/background/extension.ts

```diff
@@ -92,7 +92,12 @@ export class Extension {
 
         if (chrome.commands) {
             // Firefox Android does not support chrome.commands
-            chrome.commands.onCommand.addListener(async (command, {id: tabId}) => this.onCommand(command as Command, tabId, 0, undefined));
+            if (isFirefox) {
+                // Firefox may not register onCommand listener on extension startup so we need to use setTimeout
+                setTimeout(() => chrome.commands.onCommand.addListener(async (command) => this.onCommand(command as Command, null, null, null)));
```

Comment:
This is the actual fix for #10176, the rest is just minor style changes/nits/etc.